ui/package.json

```diff
@@ -89,7 +89,6 @@
     "d3-shape": "~3.2.0",
     "date-fns": "~2.30.0",
     "date-fns-tz": "~1.3.8",
-    "deepmerge": "~4.3.1",
```

Comment:
we didn't use this anywhere but in a path-to-try file that was not consumed anywhere.